File: sdk-python/examples/forms/move_element.py
"""Run with: python examples/forms/move_element.py"""

from __future__ import annotations

from threecommon import ThreeCommon
from threecommon.forms import MoveElementBody


def main() -> None:
    with ThreeCommon(api_key="3co_your_api_key_here") as client:
        form = client.forms.move_element(
            "frm_replace_with_real_id",
            "elm_replace_with_real_id",
            MoveElementBody(position=2),
        )
        print(f"moved element on form {form.id} ({form.status})")


if __name__ == "__main__":
    main()

File: sdk-node/examples/forms/move-element.ts
/**
 * Move an element to a new position within the form.
 *
 * Run:
 *   npx tsx examples/forms/move-element.ts
 */

import { ThreeCommon } from '@3common/sdk'

const client = new ThreeCommon({
  apiKey: '3co_your_api_key_here',
})

const form = await client.forms.moveElement(
  'frm_replace_with_real_id',
  'elm_replace_with_real_id',
  { position: 2 },
)

console.log(`moved element; form ${form.id} now has the updated layout`)
